content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -970,6 +982,12 @@ users should be aware of:
 * The `dataSourceRef` field may contain different types of objects, while the `dataSource` field
   only allows PVCs and VolumeSnapshots.
 
+In case of enabling the `CrossNamespaceVolumeDataSource` feature gate, there are three important differences between the `dataSource` field and the `dataSourceRef` field that users should be aware of: .
+
+* While the `dataSource` field only allows two specific types of objects, the `dataSourceRef` field allows any non-core object, as well as PersistentVolumeClaim objects.
+* While the `dataSource` field ignores disallowed values (dropping them), the `dataSourceRef` field preserves all values, and generates an error if a disallowed value is specified.
+* While the `dataSource`field only allows local objects, the `dataSourceRef` field allows objects in any namespaces.
```

Comment:
I think the first 2 are repeats of L979-L983.

Probably can reword this to say something like

```
When the CrossNamespaceVolumeDataSource feature is enabled, there are additional differences:
 
* The `dataSource`field only allows local objects, while the `dataSourceRef` field allows objects in any namespaces.  
* `dataSource` and `dataSourceRef` are not synced.
```

Reply:
Thank you for suggestion. Fixed https://github.com/kubernetes/website/pull/37802/commits/7cfe0a3f71a8d53f981052ab2c5864c771ce3230